config/sync/system.performance.yml

```diff
@@ -10,7 +10,7 @@ fast_404:
   exclude_paths: '/\/(?:styles|imagecache)\//'
   html: '<!DOCTYPE html><html><head><title>404 Not Found</title></head><body><h1>Not Found</h1><p>The requested URL "@path" was not found on this server.</p></body></html>'
 js:
-  preprocess: true
+  preprocess: false
```

Comment:
Change this back to true and disable JS preprocessing in your settings.local.php file. Give me a shout if you need a hand setting this up.

Reply:
Changed the JS preprocess back on.